src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,95 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_VOLUME = 16;
+    private static final double LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] vector;
+    private int volume;
```

Comment:
Let's rename variable as `size` instead of `volume`.

Reply:
Update: you can leave `volume` here
or better - use `vector.length` everywhere where you need `volume` - and remove this field

Reply:
it's must be capacity